### PR TITLE
Adding emissary to bundle input

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

*Issue #, if available:*

*Description of changes:*

The Prod ones will change to private ECR once the support is added in the next week or two, but this is just keeping in line with what we have now.

### CODE REVIEW

**Review by vincentni — APPROVED** [review]
/lgtm

**Review by TerryHowe — APPROVED** [review]
/approve

File: generatebundlefile/data/input_121.yaml [hunk 1]
```diff
@@ -23,3 +23,10 @@ packages:
         registry: public.ecr.aws/l0g8r8j6
         versions:
             - name: latest
+  - org: emissary 
+    projects:
+      - name: emissary 
+        repository: emissary-ingress/emissary
+        registry: public.ecr.aws/l0g8r8j6
+        versions:
+            - name: latest
```
Comment (lewisdiamond) [inline generatebundlefile/data/input_121.yaml:32]: Make sure to configure your IDE to always add newline at EOF.

Comment (jonahjon) [reply generatebundlefile/data/input_121.yaml:32]: Will look into that

**Pull request merged**